Classes/NodeSignalInterceptor.php

```diff
@@ -71,21 +80,4 @@ protected static function nodeCreateHiddenEnabled(NodeInterface $node): bool
     {
         return $node->getNodeType()->hasConfiguration('options.replication.structure.createHidden') && $node->getNodeType()->getConfiguration('options.replication.structure.createHidden');
     }
-
-    /**
-     * @return Replicator\NodeReplicator
-     */
-    protected static function getNodeReplicator(): NodeReplicator
-    {
-        return new Replicator\NodeReplicator();
-    }
-
-    /**
-     * @param NodeInterface $node
-     * @return array|null
-     */
-    protected static function getExcludedProperties(NodeInterface $node): ?array
```

Comment:
ist das korrekt, dass die rausfliegt?

Reply:
ich denke, weil den config key "excludeProperties" gibts in der neuen version eigentlich nicht mehr.